### ISSUE TITLE
Broken link?

### ISSUE THREAD

**Issue by rkazak** [elm/compiler#2294]
**Quick Summary:**

This url no longer seems to work,  http://elmlang.herokuapp.com/
but is referenced on,  https://github.com/elm/compile under 'Help'
to access the Elm Slack channel.

## SSCCE

```elm

```

- **Elm:** ???
- **Browser:** ???
- **Operating System:** ???

## Additional Details
This is an issue for the Elm website.

**Comment by lue-bird** [comment]
True. FYI the new way of joining is linked at https://elm-lang.org/community/slack

**Comment by lue-bird** [comment]
True. FYI the new way of joining is described at https://elm-lang.org/community/slack

**Comment by evancz** [comment]
Thank you for the report! It's updated to the better link now!

**Comment by rkazak** [comment]
And Thank you for the fix!
